galley/pkg/server/args.go

```diff
@@ -49,6 +50,12 @@ type Args struct {
 	// Maximum number of outstanding RPCs per connection
 	MaxConcurrentStreams uint
 
+	// Initial Window Size for gRPC connections
+	InitialWindowSize uint
```

Comment:
This is for grpc stream, and `InitialConnectionWindowSize` is for a connection, should set `InitialConnectionWindowSize` bigger than `InitialWindowSize ` by default, like InitialConnectionWindowSize = 16 * InitialWindowSize ?

Reply:
Updated.